doc/source/api.rst

```diff
@@ -6,6 +6,19 @@ can use to generate, consume, modify and verify in-toto metadata, as a more
 feature-rich, programmable alternative to the :doc:`command line tools
 <command-line-tools/index>`.
 
+.. note::
```

Comment:
General question: is this expected to bump up the major version? I think it is, right, since we're dropping params from the API too?

Reply:
Yes, absolutely, this will require a major version bump.